### ISSUE TITLE
Hints of a new plague? quest, sealed folder says its been looted in log but nothing in inventory

### ISSUE THREAD

**Issue by zeusmiddle** [chromiecraft/chromiecraft#2669]
### What client do you play on?

enUS

### Faction

- [X] Alliance
- [ ] Horde

### Content Phase:

- [ ] Generic
- [ ] 1-19
- [ ] 20-29
- [X] 30-39
- [ ] 40-49
- [ ] 50-59
- [ ] 60

### Current Behaviour

While there are bug reports about other issues of this quest and similar quests this has not been reported. When looting  the sealed envelope in the quest "Hints of a new plague?" the general tabbed will say you have looted it, but when you go in the inventory its not there. Going back to hand in the quests does not work because you do not have the sealed envelope.

### Expected Blizzlike Behaviour

This item should be present in the players inventory.

### Steps to reproduce the problem

1.) Talk to Quae in Hillsbrad near Go'Shek farm for the quest "Hints of a new Plague?"
2.) Follow the road leading out of Go'Shek farm until you find 5 undeads walking together, one of them is called Forsaken Courier and drops the "Sealed Folder" item
3.) KIll the Forsaken Courier and they drop "Sealed Folder". General tab says "You recieve the loot: Sealed Folder." 
4.) Sealed Folder is not present in the inventory and going back to Quae she does not allow you to finish the quest saying you do not have the Sealed Folder.

### Extra Notes

.gobject displayed nothing nearby for both npcs 

I am a lvl 25 priest.

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/c70cc855ff3d2724a0e0a3866b6a43db1f2faddb

### Operating system

Windows 10 Home

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)
- [lua-zone-debuff](https://github.com/55Honey/Acore_ZoneDebuff)

### Customizations

None

### Server

ChromieCraft

**Comment by Shineslippers** [comment]
Thank you for the report. I'm not able to reproduce bug on local AC. Idk what can cause your issue. Have you tried abandon quest and do it again? Have you checked your bags on `Sealed Folder` present? Is NPC have yellow question mark on turn in? Can you provide some screenshots?

![WoWScrnShot_122421_131144](https://user-images.githubusercontent.com/85256266/147344051-2e4cf146-9678-4341-8b57-feb669e7f022.jpg)

**Comment by zeusmiddle** [comment]
I had retried this quest about 4 times yesterday abandoning it and restarting.
For my level, I was level 35-36 while trying to complete it. 
I checked my bag for Sealed Folder on all attempts and it was not present. It was present when looting in the NPC's death inventory, the general tab told me I received the loot but it absolutely was not in my inventory, and it was gone from the death inventory of the Forsaken after I "looted" it.
My bags had open inventory slots for sure, it seems like it just disappeared from existence.
When I went to hand in the quest, the question mark was yellow, but it would not let me complete it saying `Sealed Folder ` 0/1. Same with the  


Today I retried it attempting to get screenshots... and it worked! The only thing done differently this time was where I found the courier, at the giant border wall separating Arathi from Hillsbrad, however as your screenshot shows you managed to complete the quest when it spawn by the farm... So it can't be the location that was the issue. 

I did notice when looting the `Sealed Folder` this time, a yellow text in the middle of the screen said that I had looted it, which I'm certain never happened on my attempts yesterday.
